Makefile

```diff
@@ -5,16 +5,16 @@ PKGSRC  := $(shell basename `pwd`)
 
 build:
 	cd ..; \
-    R CMD build $(PKGSRC)
+	R CMD build $(PKGSRC)
 
 install: build
 	cd ..; \
-    R CMD INSTALL $(PKGNAME)_$(PKGVERS).tar.gz
+	R CMD INSTALL $(PKGNAME)_$(PKGVERS).tar.gz
 
 check: build
 	cd ..; \
-    R CMD check $(PKGNAME)_$(PKGVERS).tar.gz --as-cran
+	R CMD check $(PKGNAME)_$(PKGVERS).tar.gz --as-cran
 
 test:
-	R -e 'library(genthat); devtools::test()'
+	Rscript tests/testthat.R
```

Comment:
`devtools::test()` doesn't exit the program with a meaningful return code.
(There might have been other reasons involved, hard to remember. I should have written a note with the motivation)